src/components/chart/_macro.njk

```diff
@@ -0,0 +1,45 @@
+{% from "components/list/_macro.njk" import onsList %}
+
+{% macro onsChart(params) %}
+    {% set supportedChartTypes = ['line', 'bar'] %}
+
+    <div
+        data-highcharts-base-chart
+        data-highcharts-type="{{ params.chartType }}"
+        data-highcharts-theme="{{ params.theme }}"
+        data-highcharts-title="{{ params.title }}"
+        data-highcharts-uuid="{{ params.uuid }}"
+        id="{{ params.uuid }}"
+    >
+        {% if params.chartType in supportedChartTypes %}
+            <figure class="ons-chart">
+                <h3 class="ons-chart__title">{{ params.title }}</h3>
```

Comment:
It would be nice to make the heading levels configurable here, in case different page types need to use them at different levels in the page hierarchy.

Reply:
Done